exe/theme-check-language-server

```diff
@@ -4,7 +4,7 @@
 require 'theme_check'
 
 if ENV["THEME_CHECK_DEBUG"] == "true"
-  $DEBUG = true
+  ThemeCheck::THEME_CHECK_DEBUG = true
```

Comment:
I think you'll get a Ruby warning here, you're not supposed to re-define constants.

You could either change to `$THEME_CHECK_DEBUG` or add module level method: `ThemeCheck.debug = true`.

Reply:
Funny story, I moved away from $THEME_CHECK_DEBUG because I got a warning that it wasn't initialized :see_no_evil:

Module level method sounds good!